src/resolve/resolved-manager.h

```diff
@@ -141,6 +141,9 @@ struct Manager {
         struct stat etc_hosts_stat;
         bool read_etc_hosts;
 
+        /* Enable/Disable IPv6 AAAA record query */
+        bool disable_aaaa_query;
```

Comment:
negative options are frowned upon in our codebase. i.e. options that literally you need to *enable* to *disable* a feature is just weird.